Widgets.go

```diff
@@ -871,6 +871,15 @@ const (
 	TreeNodeFlagsCollapsingHeader = TreeNodeFlagsFramed | TreeNodeFlagsNoTreePushOnOpen | TreeNodeFlagsNoAutoOpenOnLog
 )
 
+// TreeNodeIDV is similar to TreeNodeV but sets a ID different from the label.
+func TreeNodeIDV(id string, label string, flags int) bool {
```

Comment:
I'm not super happy about the name, but it's already a bit weird. I.e. `TreeNodeV` actually calls the `TreeNodeEx` cpp function..